src/lib.rs

```diff
@@ -72,7 +76,7 @@ unsafe extern "C" fn new_guest_cb(
     _uuid: *mut [c_uchar; 16usize],
     cb_ctx: *mut c_void,
 ) -> c_int {
-    debug!("KVMi new guest");
+    info!("KVMi new guest");
```

Comment:
Why did you change the log level? This feels like something that should be debug level for me.

Reply:
I think both of these callbacks are important even in INFO logs because they tell whether the socket connexion has been established or not.

But thinking about it, we already log if the connexion is successful later on, so I can switch them back to debug.